scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -130,7 +130,7 @@ namespace <%= namespace %> {
             NativeCheckout = new iOSNativeCheckout(State);
             #elif UNITY_ANDROID
             WebCheckout = new AndroidWebCheckout(this, client);
-            NativeCheckout = new AndroidNativeCheckout(State);
+            NativeCheckout = null;
```

Comment:
In the rest of the codebase do we need to add `if (NativeCheckout  == null)`  checks and log with a message saying coming soon?

Reply:
For `CanCheckoutWithNativePay`, it returns false as expected and when trying to call `CheckoutWithNativePay` the `PlatformNotSupported` exception is thrown.